src/shell.rs

```diff
@@ -352,7 +365,7 @@ async fn execute_command(
       .envs(state.env_vars())
       .stdout(stdout.into_raw())
       .stdin(stdin.into_raw())
-      .stderr(Stdio::inherit())
+      .stderr(stderr.clone().into_raw())
```

Comment:
Not a big deal because previously stderr was always inherited from the process anyway.